src/Api/AssetManager/AssetMediaFileApi.php

```diff
@@ -67,10 +67,10 @@ private function extractCodeFromCreationResponse(ResponseInterface $response): s
     {
         $headers = $response->getHeaders();
 
-        if (!isset($headers['Asset-media-file-code'][0])) {
```

Comment:
Actually HTTP headers are case insensitive according to [RFC 2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2). We cannot guarantee that this case will be respected (the server can respond with either of `Asset-Media-File-Code`. `asset-media-file-code`, `Asset-media-file-code` header)